guide/data-management-issues.rst

```diff
@@ -1,107 +1,127 @@
+on distributed-memory systems.
 
 Ensemble DA exacerbates this memory problem by requiring multiple copies,
 often 20-100x, of the model data to do the assimilation.
 
-DART uses the MPI libraries to distribute ensembles of model state data across
-distributed memory nodes.  For models with small amounts of data the code can be 
-compiled and run as a serial program but when compiled with MPI 
-it can scale up to 10,000s of nodes using Giga to Petabytes of memory.
+DART uses the Message Passing Interface (MPI) to distribute ensembles of 
+state and extended state data across processors and nodes in a distributed-memory system. 
+For models with relatively small memory demands, DART can be compiled and run as a serial program. 
+However, when compiled with MPI, it can scale to tens of thousands of processors 
+and handle memory requirements ranging from gigabytes to petabytes.
 
-Memory usage and internode communication time are mutually incompatible 
-items to minimize.  DART has different strategies that can be selected
-at runtime to use less memory per node at the cost of more time spent in communication
-of data between nodes, or use more memory per node and minimize communication time.
+There are three phases in DART that require access to the whole state vector:
 
-The following descriptions detail the different phases of the main assimilation
-program in DART, called ``filter``, and what options exist for memory layout
-and management.
+1. IO - reading and writing of the model state.
+2. Forward Operator (FO) computation. 
+3. Conversion of of a observation or state location from one vertical coordinate system to another.
 
+There is an inherent trade-off between memory usage and internode communication.
+DART provides multiple runtime strategies to balance this trade-off: 
+users can choose to reduce memory usage per MPI task at the cost of increased communication
+overhead, or to use more memory per node in order to minimize data transfer between MPI tasks.
+The whole state is always logically visible to all MPI tasks, but physically it may be distributed 
+across multiple tasks.
 
-Ensembles of data
------------------
 
-State data
-~~~~~~~~~~
+IO - reading and writing of the model state
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* N ensemble members times X items in the state vector, always resident.
-* 6 additional copies of X items for inflation, ensemble mean & sd, etc.
+The first N logical MPI tasks read and write the N state vectors from disk, where N is the ensemble size.
+A state vector is read into memory in a single MPI task, and then distributed across all the
+MPI tasks.  To minimize per node memory and IO contention, we recommend you assign the logical MPI tasks 
+to physical MPI tasks using a round-robin layout in the ensemble_manager_nml namelist.
 
-Observations
-~~~~~~~~~~~~
+Recommended setting for a round-robin layout in ensemble_manager_nml:
```

Comment:
Could be shortened to "Recommended setting for this mode:"

Reply:
leaving this in even thought it is repetitive to stress that layout 2 is round robin.